packages/bundler/src/types.ts

```diff
@@ -3,8 +3,29 @@ import { RollupWarning, RollupError } from 'rollup';
 export type BundlerError = RollupError;
 export type BundlerWarning = RollupWarning;
 
+// TODO: All errors could implement at least these fields?
+export type ValidationException = {
```

Comment:
Does this assume that all validators are source-based? For example, what if we determine that the test tree is too deep, something that should be done at runtime. There we won't have anything related to the source

Reply:
I mean any bundling/building errors, next we will have type checking is added and any other build-time errors